### DISCUSSION TITLE
How to use the "material_sandbox"?

### DISCUSSION THREAD

**Discussion by zealoote** [google/filament#4711] [Q&A]
Hello Guys,

I download the new releasee filament for macOS. When I open the binary file "material_sandbox" in filament/bin files. It did not work.

It shows "dyld: Library not loaded: @rpath/libsdl2.dylib". How can I fix it?

Thank you in advance!

**Comment by romainguy** [comment]
That's a bug on our end, the binary is not compiled with static libraries (that or we should provide libsdl2.dylib). Would you mind filing a bug please? Thanks!

**Comment by zealoote** [comment]
Okay! I will file a bug! Thanks for improving!
